File: src/main/java/com/ort/app/web/form/RandomKeywordForm.java
package com.ort.app.web.form;

import java.io.Serializable;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Pattern;

import org.hibernate.validator.constraints.Length;

public class RandomKeywordForm implements Serializable {

    private static final long serialVersionUID = 1L;

    @NotNull
    @Length(min = 3, max = 10)
    @Pattern(regexp = "[a-zA-Z]*")
    private String keyword;

    @NotNull
    private String message;

    public String getKeyword() {
        return keyword;
    }

    public void setKeyword(String keyword) {
        this.keyword = keyword;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: src/main/kotlin/com/ort/app/api/request/RandomKeywordForm.kt
package com.ort.app.api.request

import org.hibernate.validator.constraints.Length
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern

data class RandomKeywordForm(
    @field:NotNull
    @field:Length(min = 3, max = 10)
    @field:Pattern(regexp = "[a-zA-Z]*")
    val keyword: String? = null,

    @field:NotNull
    val message: String? = null
)